packages/pg-v5/commands/promote.js

```diff
@@ -96,6 +96,19 @@ async function run(context, heroku) {
     return cli.action.done()
   }
 
+  let promotedDatabaseDetails = await heroku.request({
+    host: host(attachment.addon),
+    path: `/client/v11/databases/${attachment.addon.id}`
+  })
+
+  if (promotedDatabaseDetails['following'] != null ){
```

Comment:
Nope, because users might want to promote a follower database, so that'd be a valid use case.

The aim of this PR is to show a warning if the user is promoting a database that's a follower, to try help them notice it earlier in case they forgot to unfollow. We see many cases where users run pg:promote thinking it'll take care of pg:unfollow too, which isn't the case, and this results in a broken app because it won't be able to write to the (still) read-only database.

Given I mean this to only be a warning, I put this at the end of the process so that the rest of the steps/requests aren't delayed by the extra time it takes to check this.

Reply:
Ah, okay. I assumed it would be rare for a customer to want a follower as their default database, but maybe that's not the case. If we wanted to get fancy, we could ask them to confirm that they want to promote a read-only follower.

This change is a good improvement over what we have, so 👍 .

Reply:
```suggestion
  if (!!promotedDatabaseDetails['following']){
```

Does "following" ever return an empty string rather than `null`? Perhaps using `!!` would be safer?

Reply:
True! It's definitely only `null` otherwise.